test/unit/test_autosubmit_helper.py

```diff
@@ -68,3 +69,51 @@ def test_get_allowed_members(mocker, ids, return_list_value, result,
     as_member_list.return_value = return_list_value
 
     assert helper.get_allowed_members(ids, as_config) == result
+
+@pytest.mark.parametrize('time, header_skip, experiment_exists',[
+        ('a000', False, False),
+        ('04-00-00', False, False),
+        ('04-00-00', False, True),
+        ('04:00:00', True, False),
+        ('04:00:00', True, True),
+],ids=['expid instead of time','wrong format hours','right format hours','fulldate wrong format',
+       'fulldate wrong format false']
+)
+def teste_handle_start_after(mocker, autosubmit_config: Callable, time: str,
+                             header_skip: bool, experiment_exists: bool):
+    """
+    function to test the function handle_start_time inside autosubmit_helper
+    """
+    expid = 'a000'
+    autosubmit_config(expid, experiment_data = {})
```

Comment:
No space between variable and dict here (only when declared as a variable out of a function call... except when there are types, I think).